Sources/eppo/EppoClient.swift

```diff
@@ -248,6 +248,12 @@ public class EppoClient {
         }
     }
 
+    /// Returns all flag configurations currently stored in the client.
+    /// - Returns: A dictionary mapping flag keys to their configurations, or nil if no configuration is loaded
+    public func getFlagConfigurations() -> [String: UFC_Flag]? {
```

Comment:
To summarize the feedback, returning a string representation of the UFC is not enough. It needs to be in a format that is easily passed to

https://github.com/Eppo-exp/eppo-ios-sdk/blob/bfbac41ead70f800090664facf02856d38ab94d7/Sources/eppo/EppoClient.swift#L52

in the `init` method. So we would at least need a constructor to generate a `UniversalFlagConfig` type from the string, the output of which could be used in `Configuration`'s constructor.

https://github.com/Eppo-exp/eppo-ios-sdk/blob/bfbac41ead70f800090664facf02856d38ab94d7/Sources/eppo/Configuration.swift#L7-L10

Reply:
Can we make the jump to `ConfigurationWire`? It seems odd to me that the ios would be used to export the config used to bootstrap itself; is the customer storing that data in a persistent store on the device?